deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -376,17 +376,6 @@ rules:
   - get
   - list
 ---
-# TODO: remove this, once https://github.com/rook/rook/issues/10141
```

Comment:
Did this extra rbac cause an issue? Or just following the comment that it can be removed?

Reply:
this is was extra files created which are not required.

Reply:
This RBAC is different from the extra CRD files. I'd suggest we revert this RBAC change to keep the change scoped just to the csv generation, unless it is needed for the csv generation. Removing this RBAC may have other side effects that we should confirm with csi team.

Reply:
no it was not extra crd file but, extra RBAC files. but anyway I'll talk with build team if we can test my local testing first and then merge